src/test/kotlin/com/gamingbacklog/api/gamingbacklogapi/testutil/models/MockResponse.kt

```diff
@@ -0,0 +1,48 @@
+package com.gamingbacklog.api.gamingbacklogapi.testutil.models
+
+import java.net.URI
+import java.net.http.HttpClient
+import java.net.http.HttpHeaders
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
+import java.util.*
+import javax.net.ssl.SSLSession
+
+class MockResponse(
+  mockBody: String,
+  mockStatus: Int
+) : HttpResponse<String> {
+  private val body = mockBody
+  private val status = mockStatus
+  override fun statusCode(): Int {
+    return status
+  }
+
+  override fun request(): HttpRequest {
+    TODO("Not yet implemented")
```

Comment:
When do we plan to implement these? Is there a ticket?

Reply:
Either way is fine, I think it would just be good to track the work for implementing the rest of the class (unless we actually don't need all the methods, in which case maybe use a different interface?)